Neos.ContentRepository.Core/Classes/DimensionSpace/DimensionSpacePoint.php

```diff
@@ -93,6 +93,11 @@ final public static function fromUriRepresentation(string $encoded): self
         return self::instance(json_decode(base64_decode($encoded), true));
     }
 
+    final public function toUriRepresentation(): string
+    {
+        return base64_encode(json_encode($this->coordinates, JSON_THROW_ON_ERROR));
```

Comment:
I don't see the benefit of base64 here.

Reply:
Yes this state is highly inefficient as concluded in https://github.com/neos/neos-development-collection/issues/5072#issuecomment-2119249935, but its nothing new but what came with the nodeAddress 6 years ago, so to shortcut the discussion i thought ill do it like this for now and followup with https://github.com/neos/neos-development-collection/pull/5067 where we can focus on this fully.

This pr is about 'bigger' things.